### ISSUE TITLE
Doesn't run on macOS

### ISSUE THREAD

**Issue by kjyv** [pepa65/tldr-bash-client#4]
Apparently macOS' bash version (3.2) is too old. Using macports or homebrew a new version can be installed in parallel but tldr doesn't refer to it. The correct shebang should be #!/usr/bin/env bash to use the proper version, also on linux.

**Comment by pepa65** [comment]
What problems does the script run into on bash 3.2?
I'll adjust the shebang in any case.

**Comment by kjyv** [comment]
/usr/local/bin/tldr: line 131: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
/usr/local/bin/tldr: line 133: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
/usr/local/bin/tldr: line 135: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
/usr/local/bin/tldr: line 137: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
/usr/local/bin/tldr: line 139: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
/usr/local/bin/tldr: line 140: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
/usr/local/bin/tldr: line 83: local: -l: invalid option
local: usage: local name[=value] ...
 tldr-bash-client version 0.33  http://github.com/pepa65/tldr-bash-client

 USAGE: tldr [option] [platform/]command

 [platform/]command:          Show page for command (from platform)

 platform is optionally one of: common, linux, osx, sunos

 option is optionally one of:
  -l, --list [platform]:      Show all available pages (from platform)
  -r, --render file:          Render a local file as tldr markdown
  -m, --markdown command:     Show the markdown source for command
  -c, --cache:                Cache all pages by downloading archive from repo
  -u, --update:               Re-download index file from repo
  -v, --version:              Version number and repo location
  [-h, -?, --help]:           This help overview

 Element styling: Title Description Example Code Value
 All pages and the index are cached locally under /Users/stefan/.local/share/tldr.
 By default, the cached copies will be freshly downloaded after 60 days.

**Comment by pepa65** [comment]
OK, I'm looking into it. Of course, I would strongly recommend installing a more current version of bash if you are able (the GPL doesn't bite as bad as Apple might imagine). But if you're developing bash applications for other OSX users, I understand you want to use what they would have.

**Comment by kjyv** [comment]
Oh no, sure I'm not using that old version. But it needs to stay installed in the location your script was pointing to because it's part of the protected system files (SIP) and can't/shouldn't be changed.   It's fixed now for anyone with a recent bash version at least 👍

**Comment by pepa65** [comment]
Allright, maybe I won't look further at bash-3.2 compatibility then. I would think people interested enough in commandline work to use tldr will probably be able to install a bash from this decade.
